files/bootstrap.sh

```diff
@@ -32,6 +32,7 @@ function print_help {
   echo "--service-ipv6-cidr ipv6 cidr range of the cluster"
   echo "--enable-local-outpost Enable support for worker nodes to communicate with the local control plane when running on a disconnected Outpost. (true or false)"
   echo "--cluster-id Specify the id of EKS cluster"
+  echo "--raid-local-disks Create a RAID-0 array of local NVMe disks for kubelet & containerd"
```

Comment:
(Assuming the implementation is correct) This feels like it should be on by default; I don't think folks should have to go the custom LT + custom AMI route to utilize those local disks.

Reply:
I think it's reasonable to enable by default.  In MNG I guess any user-data would be executed before the bootstrap script. This script would need to detect a user created instance storage setup so that we don't try to clobber their setup. That should just be an easy check to see if the disk is mounted.

Reply:
Fair; I guess we can err on the side of leaving it disabled until we get some user feedback.